src/test/resources/bundles/messageOrQuestionExample/config.json

```diff
@@ -28,7 +28,7 @@
     "questionState": {
       "name": "Question ",
       "description": "Question ",
-      "showDetailCardHeader": false,
+      "showDetailCardHeader": true,
       "color": "#0070C0",
```

Comment:
Same remark than previously. 
I propose to have only one 
"showDetailCardHeader" : true
in opfab, for automatic tests, to be sure there is no regression in the future.